tests/xarray/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+import xarray as xr
+import numpy as np
+
+
+@pytest.fixture
+def da():
```

Comment:
Could we add a quick docstring that lets future users know in prose, what data are generated, and where `da` is intended to be used (i.e. in the tests)?